Libraries/LLM/Evaluate.swift

```diff
@@ -178,24 +178,16 @@ public enum GenerateDisposition {
 public func generate(
     promptTokens: [Int], parameters: GenerateParameters, model: LLMModel, tokenizer: Tokenizer,
     extraEOSTokens: Set<String>? = nil,
-    didGenerate: ([Int]) async -> GenerateDisposition
-) async -> GenerateResult {
+    didGenerate: ([Int]) -> GenerateDisposition
+) -> GenerateResult {
```

Comment:
This would be an API change, so examine carefully!  If this is a problem I can look at doing this a different way, but I think it will be more complicated in the `ModelContainer` in order to support async actions.

Reply:
I propose doing a major version bump for this so we don't surprise anyone